README.md

```diff
@@ -108,6 +108,14 @@ let templateValues = {
   };
 ```
 
+###### Special Template Values
+
+Some template values are use by the generator, instead of by the EJS template system. These values are optional to provide, and are always prefixed with an underscore.
```

Comment:
```suggestion
Some template values are used by the generator, instead of by the EJS template system. These values are optional to provide, and are always prefixed with an underscore.
```
```suggestion
Some template values are use by the generator, instead of by the EJS template system. These values are optional to provide, and are always prefixed with an underscore.
```

Reply:
Addressed in b9d4159